client/my-sites/purchases/main.tsx

```diff
@@ -92,7 +96,65 @@ export function PurchaseCancel( {
 				purchaseId={ purchaseId }
 				siteSlug={ siteSlug }
 				getManagePurchaseUrlFor={ getManagePurchaseUrlFor }
-				purchaseListUrl={ `/purchases/subscriptions/${ siteSlug }` }
+				getConfirmCancelDomainUrlFor={ getConfirmCancelDomainUrlFor }
+				purchaseListUrl={ getPurchaseListUrlFor( siteSlug ) }
+			/>
+		</Main>
+	);
+}
+
+export function PurchaseAddPaymentMethod( {
+	purchaseId,
+	siteSlug,
+}: {
+	purchaseId: number;
+	siteSlug: string;
+} ) {
+	const translate = useTranslate();
+
+	return (
+		<Main className="purchases is-wide-layout">
+			<DocumentHead title={ translate( 'Billing' ) } />
+			<FormattedHeader
+				brandFont
+				className="purchases__page-heading"
+				headerText={ translate( 'Billing' ) }
+				align="left"
+			/>
+
+			<AddCardDetails
+				purchaseId={ purchaseId }
+				siteSlug={ siteSlug }
+				getManagePurchaseUrlFor={ getManagePurchaseUrlFor }
+			/>
+		</Main>
+	);
+}
+
+export function PurchaseCancelDomain( {
+	purchaseId,
+	siteSlug,
+}: {
+	purchaseId: number;
+	siteSlug: string;
+} ) {
+	const translate = useTranslate();
+
+	return (
+		<Main className="purchases is-wide-layout">
+			<DocumentHead title={ translate( 'Cancel domain' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Cancel Domain' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-level-change-payment-method-route)

Reply:
This is the same, just changed to sentence-case.